data/migrations/20181012081431_projects.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex, Promise) {
+	return knex.schema.createTable("Projects", function(tbl) {
+		tbl.increments("project_id");
```

Comment:
It's common practice to just call this `id` since it's a primary key on the projects table. Other tables that reference the projects table with foreign keys will use `project_id` and you don't want to get the two confused.